index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Laura's site</title>
+
+  <link href="styles/normalize.css" rel="stylesheet">
+  <link href="styles/index_style.css" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="index.html"> Home </a></li>
+        <li><a href="portfolio.html"> Portfolio </a></li>
+        <li><a href="journal.html"> Journal </a></li>
+        <li><a href="About.html"> About </a></li>
+      </ul>
+    </nav>
+    <h1> Laura's Site</h1>
+
+  </header>
+
+  <main>
+    <section>
+      <div class="left">
+        <h2> Laura Melgarejo Silva </h2>
+        <p> Colombian. physicist. Adventurer. Aspiring coder.
+        </p>
+        <h4><a href="About.html">Find more</a></h4>
+      </div>
+      <img src="assets/pics/laura_salchipapa.jpg" alt="laura_photo">
+    </section>
+    <section>
+
```

Comment:
I noticed an empty <p> tag above and assumed it was to create a visual separation.  I am assuming its the same here.  I am curious, is there a difference in the space provided by a <section> vs a <p>?  If not, I would suggest being consistent with which one you use.